app/services/catalog/submit_order.rb

```diff
@@ -36,7 +36,7 @@ def submit_order_item(item)
 
     def parameters(item)
       TopologicalInventoryApiClient::OrderParametersServiceOffering.new.tap do |obj|
-        obj.service_parameters = item.service_parameters
+        obj.service_parameters = Catalog::OrderItemSanitizedParameters.new(:order_item => item, :do_not_mask_values => true).process.sanitized_parameters
```

Comment:
small nit, can this be wrapped to ease readability?

Reply:
no prob, pulled it into a method.